stubs/docker/docker/api/container.pyi

```diff
@@ -154,10 +154,10 @@ class ContainerApiMixin:
     def rename(self, container: _Container, name: str) -> None: ...
     def resize(self, container: _Container, height: int, width: int) -> None: ...
     def restart(self, container: _Container, timeout: int = 10) -> None: ...
-    def start(self, container: _Container, *args, **kwargs) -> None: ...
+    def start(self, container: _Container) -> None: ...
```

Comment:
An exception is raised if any args or kwargs are given: https://github.com/docker/docker-py/blob/a3652028b1ead708bd9191efb286f909ba6c2a49/docker/api/container.py#L1128-L1133